src/config/index.ts

```diff
@@ -29,6 +29,8 @@ const configSchema = {
   TEST_UPDATE_MERKLE_DISTRIBUTOR_REWARDS_PARAMETERS_WITH_PROPOSAL: parseBoolean({ default: true }),
   WIND_DOWN_SAFETY_MODULE_PROPOSAL_ID: parseInteger({ default: null }),
   TEST_WIND_DOWN_SAFETY_MODULE_WITH_PROPOSAL: parseBoolean({ default: true }),
+  FUND_OPS_TRUST_PROPOSAL_ID: parseInteger({ default: null }),
```

Comment:
Have you ran mainnet fork tests and verified this proposal passed?

As a heads up, when testing this code with mainnet fork tests it might be helpful to ensure the config variable `FORK_BLOCK_NUMBER` is after block 16069909 ([block of last executed proposal](https://etherscan.io/tx/0xf98e9d0e0017fce10217477e7fab414b2210e9eebe28f450a71619ac036ad7fd)).

Reply:
all tests working!